pollbot/models/poll.py

```diff
@@ -75,6 +76,7 @@ class Poll(base):
     # Chat state variables
     expected_input = Column(String)
     in_settings = Column(Boolean, nullable=False, default=False)
+    created_from_native = Column(Boolean, nullable=False, default=False)
```

Comment:
```suggestion
    created_from_native = Column(Boolean, nullable=False, server_default='False', default=False)
```
Please add it in here as well, so the schema defined in the code is in sync to the database schema.